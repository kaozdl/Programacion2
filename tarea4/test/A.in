linealizacion
insertar_binario (30,a) (2,b) (31,c) (10,d) (40,e) (-1,z) (-10,x) .
por_niveles
linealizacion
balanceado (1,a) (2,aa) (3,aaaa) (1,abcd) (5,gg) (1,gh) (7,gty) (1,hll) (4,i) (2,j) (3,k) (1,l) (5,m) (1,n) (7,o) (2,p) .
insertar_binario .
Fin
