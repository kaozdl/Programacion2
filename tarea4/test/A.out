1>
2>
-----(-1,z)
------(-10,x)
----(40,e)
---(10,d)
--(31,c)
-(2,b)
(30,a)
0
6
3>
x 
z 
e 
d 
c 
b 
a 
4>(30,a)(2,b)(31,c)(10,d)(40,e)(-10,x)(-1,z)
5>
---(2,p)
--(7,o)
---(1,n)
-(5,m)
---(1,l)
--(3,k)
---(2,j)
(4,i)
---(1,hll)
--(7,gty)
---(1,gh)
-(5,gg)
---(1,abcd)
--(3,aaaa)
---(2,aa)
----(1,a)
6>
---(2,p)
--(7,o)
---(1,n)
-(5,m)
---(1,l)
--(3,k)
---(2,j)
(4,i)
---(1,hll)
--(7,gty)
---(1,gh)
-(5,gg)
---(1,abcd)
--(3,aaaa)
---(2,aa)
----(1,a)
4
7
7>Fin.
